usr/sbin/p11sak/p11sak.c

```diff
@@ -1908,109 +2129,266 @@ static CK_RV list_ckey(CK_SESSION_HANDLE session, p11sak_kt kt, int long_print)
                 printf(" %*s | ", CELL_SIZE, keytype);
             printf("%s\n", label);
         }
+        free(label);
+        free(keytype);
     }
 
-    ret = funcs->C_FindObjectsFinal(session);
-    if (ret != CKR_OK) {
-        printf("C_FindObjectsFinal failed (error code 0x%lX)\n", ret);
+    rc = funcs->C_FindObjectsFinal(session);
+    if (rc != CKR_OK) {
+        printf("C_FindObjectsFinal failed (error code 0x%lX: %s)\n", rc,
+                p11_get_ckr(rc));
         goto done;
     }
 
-    ret = CKR_OK;
+    rc = CKR_OK;
+
+done:
 
-    done: free(label);
-    free(keytype);
-    return ret;
+    return rc;
 }
 
-static CK_RV execute_cmd(CK_SESSION_HANDLE session, CK_SLOT_ID slot, p11sak_cmd cmd,
-        p11sak_kt kt, CK_ULONG keylength, CK_ULONG exponent, char *ECcurve,
-        char *label, char *attr_string, int long_print) {
-    CK_RV ret;
+static CK_RV confirmDestroy(char **user_input, char* label)
```

Comment:
We usually dont use camel case for internal functions, do we ?

Reply:
Changed internal function names.